pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -407,19 +410,26 @@ func clearExpectations(t *testing.T, manager *daemonSetsController, ds *apps.Dae
 }
 
 func TestDeleteFinalStateUnknown(t *testing.T) {
-	for _, strategy := range updateStrategies() {
-		manager, _, _, err := newTestController()
+	var ScheduleDSPods = "ScheduleDaemonSetPods"
```

Comment:
Rollback the feature gate once you're done

Reply:
Thanks @janetkuo. I thought about creating a function like that but it needed changes in some other tests that we are not touching as part of the current change(within the same file though). Probably, I can handle this in a separate PR for a refactor, coming to rolling back the featuregate, I think it doesn't matter as we are enabling and disabling for every test.